CONTRIBUTING.md

```diff
@@ -175,17 +138,19 @@ Test that the files you've uploaded succesfully download by running the `downloa
 script.
 If you run the script and it says `Access Denied` you may have missed step 8, but you can go back to the file and click the `Make it Public` button and try testing your download again.
 
-### General guidelines for analyses notebooks
+### Guidelines for analyses notebooks
```

Comment:
You may hate me for this, but Can we bump this and all of its children up a level? (also not spelling change)
```suggestion
## Guidelines for analysis notebooks
```